Source/WebKit/UIProcess/Extensions/Cocoa/API/WebExtensionContextAPIActionCocoa.mm

```diff
@@ -205,7 +205,7 @@
         }
     }
 
-    if (getOrCreateAction(tab.get())->presentsPopup())
+    if (getOrCreateAction(tab.get())->presentsPopup() && !popupIsOpen())
```

Comment:
This would move up to the top of this method, near the other error, and based on https://github.com/w3c/webextensions/issues/160#issuecomment-2145197651 and the recent [meeting notes](https://github.com/w3c/webextensions/blob/9ea71fcc666472b2ecc1df877cfea65211db110c/_minutes/2024-06-06-wecg.md) we agreed to throw an error in this case.